src/data/en/index.ts

```diff
@@ -332,9 +333,64 @@ export const instanceData = {
       unknownProblem:
         'Sorry, there was a problem loading the content, please try again later.',
     },
-    login: {
+    auth: {
       pleaseLogInLink: 'Please log in',
       pleaseLogInText: 'to use this feature.',
+      registerTitle: 'Register your Serlo Account',
+      recoverTitle: 'Recover your account',
+      recoveryInstructions: 'Insert and submit your email address. We will then send you an email with a reset link.',
+      verify: {
+        title: 'Verify your email',
+        instructions: 'Insert and submit your email address to verify it.',
+        alreadyDone: 'Your are logged in, so you are very likely verified already.'
+      },
+      settings: {
+        title: "Change your password",
+        instruction: 'Insert your new password.',
+      },
+      loggingOut: 'Logging you out …',
+      login: {
+        confirmAction: 'Confirm Action',
+        signIn: 'Sign in to your Account',
+        logOut: 'Log out',
+        newHere: 'Are you new here?',
+        registerNewAccount: 'Register new account',
+        forgotPassword: 'Did you %forgotLinkText%?',
+        forgotLinkText: 'forget your password'
+      },
+      fields: {
+        identifier: 'Username or Email address',
+        username: 'Username',
+        password: 'Password',
+        email: 'Email',
+      },
+      messages: {
+        1010003: 'Please confirm this action by verifying that it is you.',
+        1010001: 'Sign in',
+        1010013: 'Continue',
+        1040001: 'Register',
+        1040003: 'Continue',
+        1050001: 'Your changes have been saved! 🎉',
+        1060001: 'You successfully recovered your account. Please change your password in the next minutes.',
+        1060002: 'An email containing a recovery link has been sent to the email address you provided.',
+        1080001: 'An email containing a verification link has been sent to the email address you provided.',
+        1080002: 'You have successfully verified your email address.',
+        4000001: '%reason%',
+        4000002: '%field% is missing.',
+        4000003: 'Length must be \u003e= %expected_length%, but got %actual_length%.',
+        4000005: 'The password can not be used because %reason%.',
```

Comment:
Due to https://docs.google.com/document/d/1XtbBHgp9zarJUBQHaSA_tXtwtebaxVCFMJqeVZbVDb0/edit?disco=AAAAklr2SsA shouldn't it be?
```suggestion
        4000005: '%reason%',
```

Reply:
👍 I will commit it like this for now.
the real problem here is that I don't know where the `%reason%`-string comes from. I'll investigate this.